### PR TITLE
Fixed/alvaroeds

### PR DESCRIPTION
**Pull request by alvaroeds**
Hello, I found this error in a function when following the steps to create an nft in solana. I also found a broken URL in the main readme.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -2290,6 +2290,15 @@
...
       ]
+    },
+    {
+      "login": "alvaroeds",
+      "name": "Alvaro Enrique",
+      "avatar_url": "https://avatars.githubusercontent.com/u/41646326?v=4",
+      "profile": "https://github.com/alvaroeds",
+      "contributions": [
+        "doc"
+      ]
```
Comment (adilanchian) [inline .all-contributorsrc:2301]: you don't need to add this :). Bot will do all of this

Please remove!

File: Solana_NFTs/en/Section_3/Lesson_1_Call_Candy_Machine_From_App.md [hunk 2]
```diff
@@ -281,12 +281,18 @@ const CandyMachine({walletAddress}) => {
...
     const itemsRedeemed = candyMachine.itemsRedeemed.toNumber();
     const itemsRemaining = itemsAvailable - itemsRedeemed;
     const goLiveData = candyMachine.data.goLiveDate.toNumber();
   
+    const presale =
+      candyMachine.data.whitelistMintSettings &&
+      candyMachine.data.whitelistMintSettings.presale &&
+      (!candyMachine.data.goLiveDate ||
+        candyMachine.data.goLiveDate.toNumber() > new Date().getTime() / 1000);
+
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_1_Call_Candy_Machine_From_App.md:295]: @AlmostEfficient can I get a second pair of eyes on this please :)?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Thanks so much for the changes here!

Asking Raza to check this out for me real quick :)!


Thanks!

File: .all-contributorsrc (hunk 1 above)
Comment (alvaroeds) [reply .all-contributorsrc]: @adilanchian  Ready! I just deleted those changes :D

**Comment by helloitsm3** [comment]
Content is outdated. We've already migrated to the new Sugar CLI

**Pull request closed without merging**